update/ansible/playbook/tasks/files/datasources.yml

```diff
@@ -48,3 +48,9 @@ datasources:
   url: http://localhost:9093/alertmanager/
   jsonData:
     keepCookies: []
+- name: ClickHouse (New)
+  type: grafana-clickhouse-datasource
+  jsonData:
+    port: ${PERCONA_TEST_PMM_CLICKHOUSE_PORT}
+    server: ${PERCONA_TEST_PMM_CLICKHOUSE_HOST}
+    tlsSkipVerify: true
```

Comment:
What if the database is provisioned externally? It seems like we may need TLS.
@0leksii @BupycHuk What do you think?

Reply:
Done

Reply:
> Done

I'm curious: how did we address it?

Reply:
Ouch.... It seems that I disabled it. And then thought I forgot to switch the flag and switched it back....
Sorry for that. Will disable it.